data/formatters/generic.yaml

```diff
@@ -67,6 +67,43 @@ short_message:
 - 'performed {operation_name}'
 - 'on {resource_id}'
 ---
+type: 'conditional'
+data_type: 'azure:applicationgatewayaccess:json'
+message:
+- 'instanceId: {instance_id}'
+- 'clientIP: {client_ip}'
+- 'clientPort: {client_port}'
+- 'httpMethod: {http_method}'
+- 'originalRequestUriWithArgs: {original_request_uri_with_args}'
+- 'requestUri: {request_uri}'
+- 'requestQuery: {request_query}'
+- 'userAgent: {user_agent}'
+- 'httpStatus: {http_status}'
+- 'httpVersion: {http_version}'
+- 'receivedBytes: {received_bytes}'
+- 'sentBytes: {sent_bytes}'
+- 'clientResponseTime: {client_response_time}'
+- 'timeTaken: {time_taken}'
+- 'WAFEvaluationTime: {waf_evaluation_time}'
+- 'WAFMode: {waf_mode}'
+- 'transactionId: {transaction_id}'
+- 'sslEnabled: {ssl_enabled}'
+- 'sslCipher: {ssl_cipher}'
+- 'sslProtocol: {ssl_protocol}'
+- 'sslClientVerify: {ssl_client_verify}'
+- 'sslClientCertificateFingerprint: {ssl_client_certificate_fingerprint}'
+- 'sslClientCertificateIssuerName: {ssl_client_certificate_issuer_name}'
+- 'serverRouted: {server_routed}'
+- 'serverStatus: {server_status}'
+- 'serverResponseLatency: {server_response_latency}'
+- 'originalHost: {original_host}'
+- 'host: {host}'
+short_message:
+- 'clientIP: {clientIP}'
```

Comment:
I believe this should be:
```
- 'clientIP: {client_ip}'
```
since client_ip is the attribute name

Reply:
Similar comment for the 3 lines below this one. The value name should match the class attribute name.